src/strymon_runtime/src/lib.rs

```diff
@@ -23,16 +23,13 @@ extern crate timely_communication;
 extern crate rand;
 
 extern crate serde;
-#[macro_use]
-extern crate serde_derive;
-
-extern crate abomonation;
-#[macro_use]
-extern crate abomonation_derive;
```

Comment:
Would you mind removing these crates from the Cargo.toml as well?

Reply:
Thanks, changed in cb58534.